test/deferred-test.el

```diff
@@ -142,38 +142,38 @@
 
 (ert-deftest deferred-primitive-scope ()
   "> call-lambda lexical-scope"
-  (should= 3 (lexical-let ((st 1))
+  (should= 3 (let ((st 1))
               (deferred:call-lambda
                 (lambda () (+ st 2)))))
-  (should= 3  (lexical-let ((st 1))
+  (should= 3  (let ((st 1))
                (deferred:call-lambda
                  (lambda () (+ st 2)) 0)))
-  (should= 3 (lexical-let ((st 1))
+  (should= 3 (let ((st 1))
               (deferred:call-lambda
-                (lambda (x) (+ st 2)))))
-  (should= 3  (lexical-let ((st 1))
+                (lambda (_) (+ st 2)))))
+  (should= 3  (let ((st 1))
                (deferred:call-lambda
-                 (lambda (x) (+ st 2)) 0))))
+                 (lambda (_) (+ st 2)) 0))))
 
 (ert-deftest deferred-primitive-compile ()
```

Comment:
This test should be tested by Emacs 25 or higher version. Because Emacs 24.5 or lower versions does not support compiling closure. This causes CI-failed.